### PR TITLE
fix broken "Share this article: {link}" message strings

### PR DESCRIPTION
**Pull request by escattone**
The English `msgstr` for the `msgid` `Share this article: {link}` was somehow changed to `Was this article helpful?`, which seems to have led to a set of locales creating translations for `Was this article helpful?` rather than `Share this article: {link}`. This PR fixes two English `msgstr` values, and then clears the existing translations for the broken locales so that they can all be re-translated in Pontoon.

### CODE REVIEW

File: en_US/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -10210,12 +10210,12 @@ msgstr "Start"
 #: kitsune/wiki/jinja2/wiki/document.html:95
 #, fuzzy
 msgid "Share this article: {link}"
-msgstr "Was this article helpful?"
+msgstr "Share this article: {link}"
```
Comment (akatsoulas) [inline en_US/LC_MESSAGES/django.po:10213]: Why do we need to have `msgstr` in English? This PR will probably fix the specific problem but there are other places where the `msgstr` and the `msgid` have diverted. I am not really sure I understand the need to have msgstr for English

**Review by akatsoulas — COMMENTED** [review]
Another idea to approach this could be to reset the en_US locale and leave the `mststr` empty. Thoughts?

**Comment by escattone** [comment]
@akatsoulas After reading https://mozilla-pontoon.readthedocs.io/en/latest/user/localizing-your-projects.html#folder-structure, here's what I think we should do:

- Delete the `en_US` folder entirely. The `templates` folder is used as the "source" locale, not the `en_US` folder.
- Keep the changes to the `Share this article: {link}` `msgid` for all of the other locales.

I'll modify this PR to do the above.

**Review by akatsoulas — APPROVED** [review]
Let's this give a try in staging

**Comment by mirunacurtean** [comment]
Works for:
eu- Share this article: https://mzl.la/3SBgJcw 
fa = Share this article: https://mzl.la/3SBgJcw
hi-In = Share this article: https://mzl.la/3SBgJcw 
[lt](https://support.allizom.org/lt/kb/nmjvbghdj) = Share this article: https://mzl.la/3SBgJcw
[no](https://support.allizom.org/no/kb/sfdgh) = Share this article: https://mzl.la/3SBgJcw

I could not figure what locales would **ff** and **rm** abbreviation stand for. 
Seems like now it is the same shortlink that gets generated.

**Comment by peiying2** [comment]
> I could not figure what locales would **ff** and **rm** abbreviation stand for. Seems like now it is the same shortlink that gets generated.

`ff` is Fulah, `rm` is Romansh. Here is a list of all the communities contributing on Pontoon: https://pontoon.mozilla.org/teams/

**Pull request merged**